src/main/java/teammates/common/datatransfer/questions/FeedbackQuestionDetails.java

```diff
@@ -248,11 +269,13 @@ public void setQuestionText(String questionText) {
         this.questionText = questionText;
     }
 
-    public boolean isCommentsOnResponsesAllowed() {
+    public boolean isInstructorCommentsOnResponsesAllowed() {
```

Comment:
This one can also be abstract?

Reply:
I think it is better to not have it as abstract because default behavior is allow comments from instructor except for in contribution questions.

Reply:
Okay